src/04-modality-specific-files/08-genetic-descriptor.md

```diff
@@ -0,0 +1,60 @@
+# Genetic Descriptor
+
+The extension was primarily developped by Cyril Pernet and Clara Moreau with contributions from Tom Nichols and Jessica Turner.
```

Comment:
Do you have a "preprint" or blogpost or some other document where you describe the process a bit more in detail? It'd be cool to link it here instead of the list of names, because that would be more informative and provide more background ... akin to the publications in EEG, MEG, iEEG

Reply:
we gonna try a GigaScience editorial - but will take a little while as we also want to push a dataset